<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEmprestimos</class>
 <widget class="QDialog" name="DialogEmprestimos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bem Vindo</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetListagem1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>861</width>
     <height>221</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>19</width>
     <height>0</height>
    </size>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>127</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>41</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>42</number>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Titulo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Autor</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Editora</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ISBN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Disp. Emprestimo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Disp. Reserva</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Qtd de Livros</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ID :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditID">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonID">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRelatorio">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Relatorio</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDebito">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>159</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelNome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>118</y>
     <width>545</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTipo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetListagem2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>513</width>
     <height>221</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>19</width>
     <height>0</height>
    </size>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>127</number>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Titulo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Autor</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Editora</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ISBN</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelInf">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>574</x>
     <y>82</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pesquisar:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPesquisar">
   <property name="geometry">
    <rect>
     <x>626</x>
     <y>82</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIr">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>80</y>
     <width>35</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEmprestar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Emprestar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReservar">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reservar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDevolver">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Devolver</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
